src/printer.js

```diff
@@ -767,7 +801,7 @@ function printStatement(path, options, print) {
       case "parameter": {
         const name = concat([
           node.nullable ? "?" : "",
-          node.type ? path.call(print, "type") + " " : "",
+          node.type ? concat([path.call(print, "type"), " "]) : "",
```

Comment:
I try 1 hour on catch and fix it :imp:  When we use `path.call` it can return not string, also `object`. Don't concatenate string (i.e. `+ "string"`) - use `concat([..args])` :smile: